openmdao/recorders/sqlite_reader.py

```diff
@@ -127,6 +130,12 @@ def _load(self):
                 self.driver_cases._case_keys = [coord[0] for coord in rows]
                 self.driver_cases.num_cases = len(self.driver_cases._case_keys)
 
+                cur.execute("SELECT iteration_coordinate FROM driver_derivatives ORDER BY id ASC")
```

Comment:
per slack discussion this should be wrapped in a try/except for reading legacy databases that do not have this table  (and/or the database version should be updated due to change in schema)

Reply:
This will be addressed in the follow-on story https://www.pivotaltracker.com/story/show/159152450